compute_endpoint/globus_compute_endpoint/MULTIUSER_SETUP.md

```diff
@@ -0,0 +1,334 @@
+# Multi-User Compute Endpoints
+
+Multi-user compute endpoints (MEP) enable an administrator to securely offer
+their compute resources to their users without mandating the typical shell
+(i.e., ssh) access.  Administrators may preset endpoint configuration options
+(e.g., Torque, PBS, or Slurm), and also offer user-configurable items (e.g.,
+account id, problem-specific number of blocks).
+
+## tl;dnr
+
+For those who just want to use the tool:
+
+### Administrator setup
+
+1. Install the `globus-compute-endpoint` package from PyPI.  (These
+   instructions use a basic virtualenv but that's not mandatory; the key detail
+   is that the software must be accessible to regular user accounts, so cannot
+   be installed in, say, `/root/`.)
+
+    ```console
+    # virtualenv /opt/globus-compute-endpoint
+    # . /opt/globus-compute-endpoint/bin/activate
+    # python -m pip install globus-compute-endpoint
+    ```
+
+1. Optionally, create an alias for easier typing in this session:
+
+    ```console
+    # alias globus-compute-endpoint=/opt/globus-compute-endpoint/bin/globus-compute-endpoint
+    ```
+
+1. Create the Multi-User Endpoint configuration with the `--multi-user` flag
+   (currently not advertised in `--help`) to the `configure` subcommand:
+
+    ```console
+    # globus-compute-endpoint configure --multi-user prod_gpu_large
+    Created multi-user profile for endpoint named <prod_gpu_large>
+
+        Configuration file: /root/.globus_compute/prod_gpu_large/config.yaml
+
+        Example identity mapping configuration: /root/.globus_compute/prod_gpu_large/example_identity_mapping_config.json
+
+        User endpoint configuration template: /root/.globus_compute/prod_gpu_large/user_config_template.yaml
+        User endpoint environment variables: /root/.globus_compute/prod_gpu_large/user_environment.yaml
+
+    Use the `start` subcommand to run it:
+
+        $ globus-compute-endpoint start prod_gpu_large
+    ```
+
+1. Setup the identity mapping configuration&nbsp;&mdash;&nbsp;this depends on
+   your site's specific requirements and may take some trial and error.  The
+   key point is to be able to take a Globus Auth Identity set, and map it to a
+   local username _on this resource_&nbsp;&mdash;&nbsp;this resulting username
+   will be passed to
+   [`getpwname(3)`](https://www.man7.org/linux/man-pages/man3/getpwnam.3.html)
+   to ascertain a UID for the user.  This file is linked in `config.yaml` (from
+   the previous step's output), and, per initial configuration, is set to
+   `example_identity_mapping_config.json`.  This file has a working (if
+   useless!) configuration; please refer to the [Globus Connect Server Identity
```

Comment:
I wouldn't consider the default as a "working" configuration since no user could have an email with the `example.com` domain. If we want to state this, I suggest we modify the default configuration to use an expression that would map to the first identity in the identity set. Otherwise, I think we should explicitly say that the multi-user endpoint will not function until this file is updated.

Reply:
Have talked about of band.  How does this most recent change work for your sensibilities?

Reply:
Looks good to me. Thanks.